partner_aging/static/description/index.html

```diff
@@ -0,0 +1,48 @@
+﻿    <section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Partner Aging</h2>
+            <h3 class="oe_slogan">Interactive Aging Views with one click access to Documents.</h3>
+        </div>
+        <div class="oe_span6">
+            <div class="oe_demo oe_picture oe_screenshot">
+                    <img src="mini_screenshot.png">
+
+                <div class="oe_demo_footer oe_centeralign">Aged balance followup from a single screen!</div>
+            </div>
+        </div>
+        <div class="oe_span6">
+            <p class="oe_mt32">
+The default OpenERP Aged Partner balance report is a static PDF that is based on the difference between credits and debits, not based on documents such as Invoices and Payments.  It also does not consider unapplied payments.  This version is interactive and more complete.</p>
+            
+           <p class="oe_mt32">
+           <a href="https://launchpad.net/openerp-shared/7.0/stable/+download/partner_aging_README.pdf" target="_blank">Review Detailed Documentation</a>
+           </p>
+        </div>
+    </div>
+</section>
+
+
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <h2 class="oe_slogan">We want to hear from you!</h2>
+        <div class="oe_span6">
+            <p class="oe_mt32">
+Ursa is always looking for feedback on our contributions.</p>
+            <p class="oe_mt32">
+If you have anything you want to share with us based on your use of this module, please let us know.            </p>
+            <p class="oe_mt32">
+You can email <a href="mailto://contact@ursainfosystems.com">contact@ursainfosystems.com</a>.  
+           </p>
+            <p class="oe_mt32">
+To learn more about us and what we do, 
+visit us on the web at <a href="http://www.ursainfosystems.com">www.ursainfosystems.com</a>
```

Comment:
You should remove Ursa references in an OCA logo except from author key and the section in the README file.